contracts/BadgerVotingShare.sol

```diff
@@ -219,6 +228,17 @@ contract BadgerVotingShare {
         return (rem_badger.balanceOf(_voter) * remBadgerPricePerShare) / 1e18;
     }
 
+    /*
+        The voter can also have bremBADGER
+    */
+    function _bremBadgerBalanceOf(
+        address _voter
+    ) internal view returns (uint256) {
+        uint256 remBadgerBalance = bremBadger.totalDeposited(_voter) -
```

Comment:
Looks fine

This is just the total deposited - claimed which is the locked / unclaimed balance